src/platform/site-wide/side-nav/prop-types.js

```diff
@@ -0,0 +1,15 @@
+// Dependencies
+import PropTypes from 'prop-types';
+
+export const NavItemPropTypes = PropTypes.shape({
+  depth: PropTypes.number.isRequired,
+  description: PropTypes.string,
+  expanded: PropTypes.bool.isRequired,
+  hasChildren: PropTypes.bool.isRequired,
+  href: PropTypes.string,
+  id: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
+  order: PropTypes.number.isRequired,
+  parentID: PropTypes.string,
+  isSelected: PropTypes.bool.isRequired,
+}).isRequired;
```

Comment:
I love this, and would love to someday see our prop-types implemented this way site-wide. Maybe a little out of scope for this work though 😆

Reply:
Agreed! As long as there's a **single** source of truth for the prop types, otherwise I fear we might have the same prop type defined in multiple places in the codebase! 😬But I suppose that's how it is currently anyways lol